src/components/AvatarSkeleton.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {Circle} from 'react-native-svg';
+import themeColors from '@styles/themes/default';
+import SkeletonViewContentLoader from './SkeletonViewContentLoader';
+
+const propTypes = {
+    /**  */
+    shouldAnimate: PropTypes.bool,
+};
+
+const defaultTypes = {
+    shouldAnimate: true,
```

Comment:
Is this prop needed at all?

Reply:
Let's remove it for now. If it's needed in the future, we can implement it then.